### ISSUE TITLE
Can you put the autoaim bind back, or at least make an option to have it?

### ISSUE THREAD

**Issue by plagueXR6** [Kalaborative/survivio-plus#13]

**Comment by Kalaborative** [comment]
Okay so here would be the options:

-  I'm reversing the binds to put emotes back on B key and auto aim on right mouse click. 
-  I can make an "Always On" option so for people who prefer to not have to use two buttons, you can play with it always on
- With the *Always On* option enabled, right-clicking will by default have no effect, and you will have to change keybinds manually if you want right-clicking to do something else (like emotes)

tl;dr I can put the bind back, it's up to you to decide what you bind them to

**Comment by plagueXR6** [comment]
Yeah I think havong the always on option would cover all preferences.

**Comment by Kalaborative** [comment]
Yeah, the problem I'm having here is when trying to set it back (turning Always On off) doesn't make it go back to normal and it stays always active. The button functionality is there but it's the logic underneath that's annoying me. Hopefully I will figure out the logic...

**Comment by Kalaborative** [comment]
Added the toggle
